src/UmbracoIdentity.sln

```diff
@@ -1,7 +1,7 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 14
-VisualStudioVersion = 14.0.23107.0
+# Visual Studio 2013
```

Comment:
I'll prob revert the changes made in this file, not sure about that ProjectReferences part either.